playbooks/gpu_ml_benchmark_deploy.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Deploy Nvidia Deep Learning SSD Detection Training benchmark
+  hosts: localhost
+  connection: local
+  gather_facts: true
+  roles:
+    - role: nvidiadl_ssd_benchmark_deploy
```

Comment:
could you use the prefix gpu_benchmark` for your roles/plays and toolbox directory?
and then `mlperf` as part of the name; I guess most of it isn't specific to SSD benchmark?